src/index.html

```diff
@@ -5,9 +5,48 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img 
+        class="card__image card__image--img" 
+        src="images/imac.jpeg" 
+        alt="APPLE A1419 iMac 27&quot; Retina <br> 5K Monoblock (MNED2UA/A)"
+      >
+      <h3 class="card__title">
+        APPLE A1419 iMac 27&quot; Retina<br>5K Monoblock (MNED2UA/A)
+      </h3>
```

Comment:
Do not hardcode new line for text
```suggestion
      <h3 class="card__title">
        APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)
      </h3>
```